mobly/controllers/android_device_lib/adb.py

```diff
@@ -29,7 +29,7 @@
 
 # Number of attempts to execute "adb root", and seconds for interval time of
 # this commands.
-ADB_ROOT_RETRY_ATTMEPTS = 3
+ADB_ROOT_RETRY_ATTEMPTS = 5
```

Comment:
Can you explain why this increase is needed?
Why 5 instead of 3?

Reply:
I found empirically that for low end devices 2 retries is not enough, and on average they need 1-2 more. With 5 I noticed it succeeds 100% of the times. If it helps I can send you the results of a test where I simply keep rebooting the device and monitor the failures to reconnect after reboot.

Reply:
Applied changes as discussed